src/server/components/FilterListedComponent.jsx

```diff
@@ -5,7 +5,7 @@ import { FormGroup, filterStyles, Label } from '@admin-bro/design-system'
 
 const FilterListedComponent = props => {
   const { onChange, property, theme } = props;
-  console.log(props);
+  logger.debug(props);
```

Comment:
Removing this, I added it for debugging purposes but it's not required. This is a adminBro custom component in jsx.